Fix apolloconfigdb.sql apolloportaldb.sql AppId length settings are inconsistent (#4725)
apolloconfig/apollo: CHANGES.md

Apollo 2.2.0

------------------

> * [Fix apolloconfigdb.sql apolloportaldb.sql appId length settings are inconsistent](https://github.com/apolloconfig/apollo/pull/4725)

Reviewer: ```suggestion
* [Fix the problem of inconsistent length of appId column](https://github.com/apolloconfig/apollo/pull/4725)
```

* [Portal-UI adds serverConfig configuration management of ApolloConfigDB](https://github.com/apolloconfig/apollo/pull/4680)
* [Enable login authentication for eureka](https://github.com/apolloconfig/apollo/pull/4663)
* [Add github action to publish docker image](https://github.com/apolloconfig/apollo/pull/4685)

> * [Fix apolloconfigdb.sql apolloportaldb.sql AppId length settings are inconsistent](https://github.com/apolloconfig/apollo/pull/4725)

Reviewer: As #4721 is merged, please add the changes to the new CHANGES.md file.

Author: OK, I have updated the changes to the new CHANGES.md file.
